### PR TITLE
Also inject CSS files into .mustache templates

### PR DESCRIPTION
**Pull request by kraftwer1**

### CODE REVIEW

**Comment by johanneswuerbach** [comment]
Thank you for this PR, could you add another test verifying that testem still works without specifying `css_files` and squash your commits? Thank you

**Comment by kraftwer1** [comment]
Since no `css_files` are added in any of the tests and they're all running, I guess it's clear that "Testem still works without specifying" them. I have, however, added another test that makes sure CSS files are loaded correctly. Or do I misunderstand what you want? Let me know what you think. If everything is okay, I'll squash my commits.

My local tests run, but Travis fails. Any idea why?

**Comment by johanneswuerbach** [comment]
0.12 fails currently on `getTemplateData`, no idea why but not related to your changes.

Tests looks, I'll merge when squashed and windows is green.

**Comment by kraftwer1** [comment]
Great. Commits are squashed.

**Comment by johanneswuerbach** [comment]
thanks, hopefully appveyor is catching up soon.

File: tests/config_tests.js [hunk 1]
```diff
@@ -318,6 +318,20 @@ describe('Config', function(){
...
 		})
 	})
+
+	describe('getCSSFiles', function(){
+		it('loads css_files correctly', function(done){
+			config.set('cwd', 'tests')
+			config.set('src_files', 'fixtures/styles/*.css')
+			config.getSrcFiles(function(err, files){
+				expect(files).to.deep.equal([
+					fileEntry('fixtures/styles/print.css'),
```
Comment (johanneswuerbach) [inline tests/config_tests.js]: This fails on windows. Using `path.join` should fix it.

**Comment by kraftwer1** [comment]
Ready!

**Pull request merged**